tests/queries/0_stateless/03202_sz3_codec.sql

```diff
@@ -108,6 +108,12 @@ AND
     c2.key = c1.key - 1
 LIMIT 10;
 
+SELECT
```

Comment:
The exact storage sizes will probably be different on different platforms (x86 and ARM in CI) but let's wait for CI to come back.

Reply:
(Not sure, I guess a more robust test is to check the output of `DESCRIBE TABLE` (it contains a codec field))

Reply:
In any case, please add a brief code comment what the test is supposed to test.